ennead/templates/dialogue.html

```diff
@@ -0,0 +1,65 @@
+{% extends 'base.html' %}
+
+{% block title %}{{task.order_num}}. {{ task.name }}{% endblock %}
+
+{% block body %}
+	<div class="row">
+		<div class="col-4 offset-4">
+			<h3>{{ task.order_num }}. {{ task.name }}</h3>
+			<p class="score">Балл: {{ thread.score }} из {{ task.base_score }}</p>
+			<div class="task-description">{{ task.html_description|safe }}</div>
+			{% if g.user.is_teacher %}
+				<div class="task-solution">{{ task.solution_html_description|safe }}</div>
+			{% endif %}
+		</div>
+	</div>
+	{% for post in posts: %}
+		{% if post.author.is_teacher %}
+			{% set offset_class = 'offset-1' %}
+			{% if post.hide_from_student %}
+				{% set post_klass = 'post-teacher-hidden' %}
+			{% else %}
+				{% set post_klass = 'post-teacher' %}
+			{% endif %}
+		{%else%}
+			{% set offset_class = '' %}
+			{% set post_klass = 'post-student' %}
+		{%endif%}
+		<div class="row">
+			<div class="col-11 {{offset_class}}">
+				<div class="post {{post_klass}} ">
+					<div class="post-header">
+						{% if g.user.is_teacher %}
+							<div class="post-author">{{ post.author.username }}</div>
+						{% endif %}
+						<div class="post-date">{{ post.date.strftime('%d %B, %H:%M') }}</div>
+					</div>
+					<div class="post-body">{{ post.html_description|safe }}</div>
+				</div>
+			</div>
+		</div>
+	{% endfor %}
+	<div class="row">
+		<div class="col-8 offset-2">
+			<hr/>
+			<form id="post-form" action="{{ url_for('post_to_thread', task_id=task.id, student_id=student.id) }}" method="POST">
+				<textarea name="text" placeholder="Решение задачи" id="post-textarea"></textarea>
+				{% if g.user.is_teacher: %}
```

Comment:
```suggestion
				{% if g.user.is_teacher %}
```
Проверь в других местах, нет ли такой же опечатки